admin/auth_token.go

```diff
@@ -177,6 +177,7 @@ type IssueMagicAuthTokenOptions struct {
 	Fields          []string
 	State           string
 	Title           string
+	Internal        bool
```

Comment:
Also just a though – do you think we should make this a `Type string` so we can also use these magic auth tokens for other use cases in the future (like alerts)? For example, we could have `type=public_url` for the previous usage, and `type=report_export` for the new use case.

Reply:
That can be done however wouldn't we always need one more table for storing mapping from resource name and recipient to magic tokens like we have for report tokens ? So storing type will be redundant ?